tools/enso4igv/package.json

```diff
@@ -52,6 +52,12 @@
         "configuration": "./src/main/resources/org/enso/tools/enso4igv/enso.tmLanguage.json"
       }
     ],
+    "netbeans.documentSelectors": [
```

Comment:
Register support for Enso language via https://github.com/apache/netbeans/pull/6174 API that will be available in post 18.0.0 release of the [extension](https://marketplace.visualstudio.com/items?itemName=ASF.apache-netbeans-java).